quickstart/docker/minimal/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM debian:bookworm-slim
```

Comment:
why isn't this extending ziti-cli? seems like it should be?

FROM ${ZITI_CLI_IMAGE}:${ZITI_CLI_TAG}

Reply:
The purpose of this Dockerfile is to support a secondary use case for the minimal quickstart: local development environment running the local build of the `ziti` executable from the `/build/ziti` path at the top-level of the checked-out Git working copy.

This means the Dockerfile is only used for running the local build of `ziti`, and has no relationship to the `ziti-cli` container image, which is used for delivering CI builds of `ziti`.

The only requirements for this Linux container image are that it has a small footprint and it has a version of GLIBC that is fairly new, because the `zrok` binary that is built locally will be linked to the `libc` installed on the developer's workstation. The `ziti-cli` image, in contrast, has a `ziti` binary that is intentionally linked to an older `libc` for broader compatibility. This means the locally-built `ziti` binary will likely not be executable inside the `ziti-cli` container image.